blockchain/vm/jump_table.go

```diff
@@ -60,14 +61,20 @@ type operation struct {
 	returns bool // determines whether the operations sets the return data content
 }
 
-// TODO-IncompatibleChange: define new InstructionSet for first incompatible change
 var (
 	ConstantinopleInstructionSet = newConstantinopleInstructionSet()
+	IstanbulInstructionSet       = newIstanbulInstructionSet()
 )
 
 // JumpTable contains the EVM opcodes supported at a given fork.
 type JumpTable [256]operation
 
+func newIstanbulInstructionSet() JumpTable {
```

Comment:
So this shouldn't be used until we add Istanbul hardfork items, right?

Reply:
If so, how about adding a Crit logger here to prevent the misuse of the function?

Reply:
@ehnuje This function itself is okay. Looking up a few lines, you can see a variable `IstanbulInstructionSet` that contains the actual instructionSet which is initialized by this function. However, this variable is never used, because there is no code that uses it. I will put crit when the code that uses it is entered :)